doc/source/overview/dev_practices.rst

```diff
@@ -194,4 +194,372 @@ should not wait until a minor release. These are the steps for a patch release:
 
 Testing
 -------
-<Testing Section>
+Unit testing is critical for the sucessful continious integration and testing of
+any program or libraries belonging to the PyAnsys project are no exception.
+
+There are generally two types of libraries part of the PyAnsys project: those
```

Comment:
```suggestion
The PyAnsys project generally has two types of libraries: those
```

Reply:
These two might best be described in a bulleted list.

Reply:
@PipKat Great idea. Otherwise the sentence is a little bit difficult to read. I am doing it.